### ISSUE TITLE
highstock exportChart 5.0.6 & 5.0.7 works ok only the first time, when zoomed in. 5.0.5 is ok

### ISSUE THREAD

**Issue by mhwlng** [highcharts/highcharts#6319]
I updated to highstock 5.0.7

When I 'zoom in' to a smaller date range and call exportChart, all is ok.

When I now call exportChart again, for the second time, then the resulting svg string is for the 'full date range' instead.
No longer for the zoomed in chart, that you can see on the screen.

Reverting exporting.js to 5.0.5 fixes the problem.

The problem exists in both 5.0.7 and 5.0.6 exporting.js (they are identical)

The only difference with 5.0.5 exporting.js is a few lines related to axes in the 'getSVG' function

**Comment by pawelfus** [comment]
Could you recreate issue in jsFiddle? Our demos are fine, for example: http://www.highcharts.com/stock/demo/data-grouping

**Comment by mhwlng** [comment]
I fixed the problem in exporting.js 5.0.7 this way :

old :

```
each(chart.axes, function(axis) {
  axis.userOptions.internalKey = H.uniqueKey();
});

```
new :

```
each(chart.axes, function (axis) {
  if (axis.userOptions.internalKey === undefined)
    axis.userOptions.internalKey = H.uniqueKey();
});

```

AND  CHANGED OLD :

`return copy.options.internalKey === axis.userOptions.internalKey;`

TO NEW :

`return copy.userOptions.internalKey === axis.userOptions.internalKey;`

**Comment by mhwlng** [comment]
here you can reproduce it

http://jsfiddle.net/doez51u6/

1) drag to the left in range selector
2) click download svg image 
3) svg looks ok
4) click download svg image  again
5) now svg is different (wrong) it goes back to the default setting

**Comment by pawelfus** [comment]
Thank you for the demo!

**Comment by pawelfus** [comment]
It's fixed in 5.0.10. Please reopen this issue if this will show up again.
